src/module.c

```diff
@@ -662,6 +662,19 @@ void RM_SetModuleAttribs(RedisModuleCtx *ctx, const char *name, int ver, int api
     ctx->module = module;
 }
 
+/* Return non-zero if the module name is busy.
+ * Otherwise zero is returned. */
+int RM_IsModuleNameBusy(const char *name) {
+    sds modulename = sdsnew(name);
+
+    /* Check if the module name is busy. */
+    if (dictFind(modules,modulename) != NULL) {
+        sdsfree(modulename);
+        return 1;
+    }
+    return 0;
```

Comment:
No sdsfree here? Just asking.

Reply:
There is a memory leak indeed. Fixing, thanks.

Reply:
Fixed